# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../authkit/egginfo-elementtree.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- AuthKit-0.4.3/AuthKit.egg-info/requires.txt.orig	2009-01-23 14:32:31.000000000 +0200
+++ AuthKit-0.4.3/AuthKit.egg-info/requires.txt	2009-01-23 14:34:15.000000000 +0200
@@ -3,7 +3,6 @@
 PasteDeploy>=1.1
 PasteScript>=1.1
 python-openid>=2.1.1
-elementtree>=1.2,<=1.3
 Beaker>=0.7.3
 decorator>=2.1.0
 WebOb>=0.9.3
@@ -23,4 +22,4 @@
 TurboKid==0.9.5
 
 [pylons]
-Pylons>=0.9.5,<=1.0
\ No newline at end of file
+Pylons>=0.9.5,<=1.0
